api/src/main/java/io/strimzi/api/kafka/model/common/template/DnsPolicy.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model.common.template;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+public enum DnsPolicy {
+    CLUSTERFIRST,
+    CLUSTERFIRSTWITHHOSTNET,
```

Comment:
Can we use the "_" in the enum values to separate works, so having CLUSTER_FIRST and CLUSTER_FIRST_WITH_HOST_NET?